iBudget/utils/aws_helper.py

```diff
@@ -0,0 +1,66 @@
+"""
+File Handler
+Provides methods for upload and download processes of the object to Amazon S3 service
+"""
+import boto3
+
+from ibudget.settings import AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY, \
+    AWS_STORAGE_BUCKET_NAME
+
+BOTO_S3 = boto3.resource(
```

Comment:
This is not a pythonic way of using constants like this. We need a method that initiates S3 client instead of constant `BOTO_S3`. This comment is applicable for all constants in this file.

Reply:
And in general - this is a bad way of doing things. You need not a helper but something like awsService class that has methods line get_s3_client, get_file_url, get_bucket etc.

Reply:
Thanks for remark!

I will create a class in the module aws_helper and move all these methods and necessary attributes there.